external/sentry.go

```diff
@@ -0,0 +1,31 @@
+package external
+
+import (
+	"github.com/getsentry/sentry-go"
+	log "github.com/sirupsen/logrus"
+	"golbat/config"
+	"time"
+)
+
+const sentryDsn = "https://bc0b5dc992ee486a90faa20b83016351@sentry.mew.is/2"
```

Comment:
Is this the unownhash shared instance? README should be updated to make clear what is going on.

Reply:
Yes it is. Have pending README and `config.toml.example` changes, soon:tm:

Reply:
Logging to Sentry

The unown has developers have installed a Sentry instance to allow us to track performance and errors in Golbat. This contains no personal data nor anything that identifies your installation (I assume this is true?)
If you wish to disable it include:

```[sentry]
enable=false
```
in your config.toml

Reply:
```
	// If this flag is enabled, certain personally identifiable information (PII) is added by active integrations.
	// By default, no such data is sent.
	SendDefaultPII bool
```
No PII is sent - https://docs.sentry.io/platforms/go/configuration/options/